<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controlleur</class>
 <widget class="QMainWindow" name="Controlleur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controlleur</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton,QPlainTextEdit{
    text-align: center;
	   margin:5px;
    font-weight: bold;
    padding: 10px 0 10px 10px ;
    background-color: lightgray;
    color: gray;
	 border-radius: 15px;
    }
    QPushButton:hover{
    text-align: center;
     margin:5px;
    font-weight: bold;
    padding: 10px 0 10px 10px ;
    background-color: gray;
    color: lightgray;
   border-radius: 20px;
    }

QLabel {
	color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>320</y>
      <width>501</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="rightMargin">
      <number>40</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton1">
       <property name="font">
        <font>
         <family>Showcard Gothic</family>
         <pointsize>25</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Jouer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton2">
       <property name="font">
        <font>
         <family>Showcard Gothic</family>
         <pointsize>25</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Paramètres</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton3">
       <property name="font">
        <font>
         <family>Showcard Gothic</family>
         <pointsize>25</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>À propos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton4">
       <property name="font">
        <font>
         <family>Showcard Gothic</family>
         <pointsize>25</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>700</y>
      <width>971</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;Copyright © 2021. Ecole Nationale Des Sciences De l'Inforamtique.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>1501</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>491</width>
      <height>239</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelJoueur1">
         <property name="font">
          <font>
           <family>Showcard Gothic</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Joueur 1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="joueur1">
         <property name="font">
          <font>
           <family>Showcard Gothic</family>
           <pointsize>25</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="plainText">
          <string>Joueur 1</string>
         </property>
         <property name="centerOnScroll">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelJoueur2">
         <property name="font">
          <font>
           <family>Showcard Gothic</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Joueur 2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="joueur2">
         <property name="font">
          <font>
           <family>Showcard Gothic</family>
           <pointsize>25</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="plainText">
          <string>Joueur 2</string>
         </property>
         <property name="centerOnScroll">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton4</sender>
   <signal>clicked()</signal>
   <receiver>Controlleur</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>672</x>
     <y>676</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
